reference/runners.rst

```diff
@@ -0,0 +1,17 @@
+.. _reference_runners:
+
+Runners
+=======
+
+Runners are the way to run Conan remotely and transparently just by modifying the host profile. There are three requirements to be able to use the feature:
+
```

Comment:
I think this is too vague, it could be anything, could we be more specific in what the feature does to introduce it?

Reply:
Something like:

_Runners provide a seamless method to execute Conan on remote build environments like Docker ones, directly from your local setup by simply configuring your host profile._ 

maybe more elaborated...